### DISCUSSION TITLE
How to adjust text labels in previousnext extension?

### DISCUSSION THREAD

**Discussion by tschurlo** [datenstrom/community#635] [Ask a question] [answered]
Hi all!

I just installed Previousnext 0.8.7 and tested it on my blog.
In general it works well, but I have issues with the PreviousnextStyle parameter.

It's not specified further in the documentation, but I assumed, that setting it to "simple" in yellow-system.ini would lead to links just named "Previous" and "Next" instead of still including the whole blog page title. 
Is my assumption correct? 
What could I possibly have done wrong?

Kind regards,
M-Th.

**Comment by markseuffert** [comment, marked as answer]
You can ignore the setting `PreviousnextStyle`. I think it's a leftover from the beginning. It doesn't have any purpose anymore and will be removed with the next update. Text labels can be configured in the language settings  in file `system/extensions/yellow-language.ini`. Here's an example for the [previousnext extension](https://github.com/annaesvensson/yellow-previousnext):

```
Language: en
PreviousnextPagePrevious: Previous
PreviousnextPageNext: Next
```

Hope this helps.

**Reply by tschurlo** [reply]
Ok, thank you, yes, that helped.
I additionally added the arrow symbols used in the blog posts, to match the look & feel:
```
PreviousnextPagePrevious: ← Previous
PreviousnextPageNext: Next →
```
